### PR TITLE
Improve `errno` stubs on Windows and MacOS

### PR DESCRIPTION
**Pull request by AlexWaygood**

### CODE REVIEW

**Review by hauntsaninja — COMMENTED** [review]
Thanks for doing this, looks reasonable. Nit: can we coalesce the similar conditions and unflatten the nested ones?

**Comment by AlexWaygood** [comment]
> Thanks for doing this, looks reasonable. Nit: can we coalesce the similar conditions and unflatten the nested ones?

We can, yes! Mainly did it this way to make the diff more readable, but I agree that it doesn't make the end result particularly readable. I'll make the change :)

File: stdlib/errno.pyi [hunk 1]
```diff
@@ -119,40 +86,135 @@ EHOSTUNREACH: int
...
+    EPROCLIM: int
+    EPROCUNAVAIL: int
+    EPROGMISMATCH: int
+    EPROGUNAVAIL: int
+    EPWROFF: int
+    ERPCMISMATCH: int
+    ESHLIBVERS: int
+
+    if sys.platform != "darwin" or sys.version_info >= (3, 11):
+        EQFULL: int  # undocumented
```
Comment (AlexWaygood) [inline stdlib/errno.pyi:117]: Not possible to unflatten this nested check without triggering flake8-pyi's Y022

**Review by hauntsaninja — APPROVED** [review]

**Comment by hauntsaninja** [comment]
Thank you!

**Pull request merged**